apps/remote_control/lib/lexical/remote_control/build/error.ex

```diff
@@ -17,6 +37,70 @@ defmodule Lexical.RemoteControl.Build.Error do
     )
   end
 
+  defp normalize(%Result{} = result) do
+    result
+  end
+
+  defp format(%Result{} = result) do
+    %Result{result | message: format_message(result.message)}
+  end
+
+  defp format_message("undefined" <> _ = message) do
+    # All undefined messages explain the *same* thing inside the parentheses,
+    # like: `undefined function print/1 (expected Foo to define such a function or for it to be imported,
+    #        but none are available)`
+    # that makes no sense and just creates noise.
```

Comment:
I disagree strongly that this makes no sense. It's a helpful detail message that can indicate something isn't working like you would expect. 
I get a little concerned about the maintainability of these fixup functions, they're very dependent on messages being consistent from release to release.

I would suggest omitting them.

Reply:
<img width="1094" alt="image" src="https://github.com/lexical-lsp/lexical/assets/12830256/c61adce3-9de3-4ccd-8a30-c1aee87842da">
You prefer the above, right?